docs/README.md

```diff
@@ -28,6 +28,18 @@ Types like "bar" and "line" support [axis rotation](http://c3js.org/samples/axes
 ### Custom (built by us):
 
 * metric
+
+Will display results and if `previousResults` are set, show the difference between them.
```

Comment:
If `previousResults` is set, the metric will show the difference between the current result and the previous one.